spring-cloud-starter-dataflow-metrics-collector/src/main/java/org/springframework/cloud/dataflow/metrics/collector/MetricCollectorProperties.java

```diff
@@ -23,7 +23,7 @@
  */
 @ConfigurationProperties(prefix = "spring.cloud.dataflow.metrics.collector")
 public class MetricCollectorProperties {
-	private Integer evictionTimeout = 30;
+	private Integer evictionTimeout = 70;
```

Comment:
Will change to 90 on merge to be 1.5x the default 60second publish interval from boot.